### DISCUSSION TITLE
Terminal In Example (README)

### DISCUSSION THREAD

**Discussion by rodoabad** [starship/starship#1958] [Q&A] [answered]
Hey folks, I have starship installed but I was wondering how do you achieve the same colour or theme just like the one in README?

Right now my terminal looks like this.

<img width="614" alt="Screen Shot 2020-12-04 at 12 27 24 PM" src="https://user-images.githubusercontent.com/1240667/101200575-16911580-362c-11eb-8812-4ff3c83ba0aa.png">

Just looking for guidance.

<img
  src="https://raw.githubusercontent.com/starship/starship/master/media/demo.gif"
  alt="Starship with iTerm2 and the Snazzy theme"
/>

**Comment by rodoabad** [comment, marked as answer]
I think I found the answer.

https://github.com/sindresorhus/terminal-snazzy

Thank goodness for accessibility!

>   alt="Starship with iTerm2 and the Snazzy theme"

**Reply by chipbuster** [reply]
While it's not easy to search for, this information can also be found in [our FAQ](https://starship.rs/faq/#what-is-the-configuration-used-in-the-demo-gif)
